instance/models/instance.py

```diff
@@ -207,52 +205,61 @@ def _sort_log_entries(server_logs, instance_logs):
         next_server_log_entry = partial(next, server_logs, None)
         next_instance_log_entry = partial(next, instance_logs, None)
 
-        log = []
         instance_log_entry = next_instance_log_entry()
         server_log_entry = next_server_log_entry()
 
         while instance_log_entry is not None and server_log_entry is not None:
             if server_log_entry.created < instance_log_entry.created:
-                log.append(server_log_entry)
+                yield server_log_entry
                 server_log_entry = next_server_log_entry()
             else:
-                log.append(instance_log_entry)
+                yield instance_log_entry
                 instance_log_entry = next_instance_log_entry()
 
         while instance_log_entry is not None:
-            log.append(instance_log_entry)
+            yield instance_log_entry
             instance_log_entry = next_instance_log_entry()
 
         while server_log_entry is not None:
-            log.append(server_log_entry)
+            yield server_log_entry
             server_log_entry = next_server_log_entry()
```

Comment:
Actually pylint does support python 3.5 now, so we could make the switch. See https://github.com/open-craft/opencraft/pull/6